packages/esm-appointments-app/src/appoinments-tabs/appointments-base-table.component.tsx

```diff
@@ -31,7 +31,7 @@ import styles from './appointments-base-table.scss';
 import { MappedAppointment } from '../types';
 import { launchOverlay } from '../hooks/useOverlay';
 import AppointmentDetails from '../appointment-details/appointment-details.component';
-import AppointmentForm from '../appointment-forms/appointment-form.component';
+import AppointmentForm from '../appointment-forms/edit-appointment-form.component';
```

Comment:
Just noticing the file name change.. is AppointmentForm only for edit? or can it be used for adding forms also?